x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -159,6 +160,12 @@ export const basicCase: Case = {
   subCaseIds: [],
 };
 
+export const basicResolvedCase: ResolvedCase = {
```

Comment:
Feel free to ignore, but maybe `aliasMatchCase` may be more descriptive?

Reply:
there is a reason why I named it _basic_ instead _aliasMatch_, the TLDR is that it is more intuitive to understand in the tests cases implementations